src/lib/uTop.mli

```diff
@@ -210,6 +210,15 @@ val history_file_max_entries : int option ref
       default) the maximum number of entries if [history] will be
       used. *)
 
+val stashable_session_history : LTerm_history.t
+  (** A history consisting only of expressions successfully evaluated
+   * during the current session. Because stashing is supposed to
+   * produce a valid OCaml file which will behave roughly the same as
+   * the console, it is best if this history never gets truncated, so
+   * its maximum size is set at [max_int]. While this will certainly
+   * lead to a slight memory leaking problem, UTop sessions are
+   * rarely long enough to make it a serious issue. *)
```

Comment:
Changed, though I'm not sure what the column limit is in uTop code. I'm familiar with the 80 column rule, but most of the existing comments seem to stop well before then.